metrics/heartbeat_test.go

```diff
@@ -116,5 +116,5 @@ func TestSetNoHeartbeatURLState(t *testing.T) {
 func TestValidateHeartbeatConfig(t *testing.T) {
 	err := ValidateHeartbeatConfig()
 	assert.Nil(t, err)
-	assert.Equal(t, false, isNoHeartbeatURL)
+	assert.Equal(t, true, isNoHeartbeatURL)
```

Comment:
Is this expected? We are changing the behaviour of the Testcase?

Reply:
Hi @raam, yes we have changed the default behavior to not have a service heartbeat